TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -78,7 +78,7 @@ private SelectClause CreateSelectClause(CodeElementsParser.Sql_selectClauseConte
             return null;
         }
 
-        private DotStarSelection CreateDotStarSelection(CodeElementsParser.SelectionContext context)
+        private TableOrViewAllColumnsSelection CreateDotStarSelection(CodeElementsParser.SelectionContext context)
```

Comment:
As this method will evolve to handle other type of selection, you should rename it `CreateSelection`. `CreateDotStarSelection` is not relevant anymore.